.github/workflows/issue-lifecycle.yml

```
@@ -0,0 +1,192 @@
+  issues:
+    types: [labeled, unlabeled, closed]
+  issue_comment:
+    types: [created]
+
+jobs:
+  manage-state-transitions:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: read
+      projects: write
```

actionlint (1.7.7)

unknown permission scope "projects". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)